serenata_toolbox/chamber_of_deputies/reimbursements.py

```diff
@@ -19,9 +18,15 @@ class Reimbursements:
     Get an updated version of the reimbursements dataset for a given year.
     """
 
-    def __init__(self, year, path=None):
+    def __init__(self, year='2018', path=None):
         self.year = year
-        self.path = path or gettempdir()
+        if not os.path.isdir(path):
+            try:
+                os.mkdir(os.path.join(path))
+            except OSError:
+                log.error('Couldn\'t create the output directory')
+                raise
```

Comment:
We should be explicit about which exception we're raising here. We can either throw the default `OSError` or build our custom exception.

TBH I would simply and just trust the `OSError`:

```python
if not os.path.isdir(path):
    os.mkdir(os.path.join(path))
```

With no `try`/`exception` at this point. What do you think about it?